### PR TITLE
add aggregation summary for multi exposure

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix?
* Ticket: #4439 
* Funded by: DFAT
* Description: add aggregation summary for multi exposure

### Checklist:
- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

File: safe/gis/vector/summary_5_multi_exposure.py [hunk 1]
```diff
@@ -148,38 +155,81 @@ def multi_exposure_analysis_summary(analysis, intermediate_analysis):
...
-    :param analysis: The target vector layer where to write statistics.
-    :type analysis: QgsVectorLayer
+    :param aggregation: The target vector layer where to write statistics.
+    :type aggregation: QgsVectorLayer
 
-    :param callback: A function to all to indicate progress. The function
-        should accept params 'current' (int), 'maximum' (int) and 'step' (str).
-        Defaults to None.summary_5_multi_exposure.py
-    :type callback: function
+    :param intermediate_layers: List of aggregation layer for a single exposure
```
Comment (ismailsunni) [inline safe/gis/vector/summary_5_multi_exposure.py]: point in the end of sentence :)

File: safe/gis/vector/tools.py [hunk 2]
```diff
@@ -302,15 +302,24 @@ def read_dynamic_inasafe_field(inasafe_fields, dynamic_field):
     :param dynamic_field: The dynamic field to use.
     :type dynamic_field: safe.definitions.fields
 
+    :param black_list: A list of fields which are conflicting with the dynamic
```
Comment (ismailsunni) [inline safe/gis/vector/tools.py]: Honestly, I am getting lost of these fields (especially the new one). Perhaps, we can make a diagram or something to make people easy to understand.

Comment (Gustry) [reply safe/gis/vector/tools.py]: True. I don't like this blacklist hack too but now I don't see any quick solution. It's getting complex day after day

File: safe/gis/vector/summary_5_multi_exposure.py (hunk 1 above)
Comment (Gustry) [reply safe/gis/vector/summary_5_multi_exposure.py]: Fill fix in my other

**Pull request merged**